app/modules/import-export/module.php

```diff
@@ -89,13 +89,11 @@ public function __construct() {
 		}
 
 		( new Usage() )->register();
+
+		$this->revert = new Revert();
 	}
 
 	public function get_init_settings() {
-		if ( ! Plugin::$instance->app->is_current() ) {
-			return [];
-		}
-
```

Comment:
why?

Reply:
Because we need the 'lastImportedSession' in the admin

Reply:
yeah, but won't it expose all of the IE data even when not needed?

Reply:
it's exposed on the admin side now too, and it's needed there now